Example/Example/ComicsViewController.swift

```diff
@@ -19,6 +19,12 @@ class ComicsViewController: ExampleViewController, BothamCollectionViewControlle
     override func viewDidLoad() {
         collectionView.accessibilityLabel = "Comics CollectionView"
         collectionView.delegate = self
+        dataSource.reuseIdentifierForItem = { item in
+            if case .Featured = item {
+                return "FeaturedComicCollectionViewCellReuseIdentifier"
+            }
+            return "ComicCollectionViewCellReuseIdentifier"
+        }
```

Comment:
Then we should extract this binding to another class. If we want to reuse this configuration in another view controller. For this case is not needed, but in the future we will want to reuse this configuration in different table views in the same app.